### PR TITLE
wasm: add kuasar-wasm systemd service

### PR DESCRIPTION
**Pull request by jwcesign**
wasm: add kuasar-wasm systemd service

### CODE REVIEW

**Comment by Burning1020** [comment]
Please rebase your commits.

**Review by Burning1020 — APPROVED** [review]
lgtm

File: wasm/service/kuasar-wasm.service [hunk 1]
```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description=Kuasar Wasm sandboxer daemon process
+After=network.target
+
+[Service]
+Type=simple
+ExecStart=/usr/local/bin/wasm-sandboxer --listen /run/wasm-sandboxer.sock --dir /run/kuasar-wasm
+ExecReload=/bin/kill -s HUP $MAINPID
```
Comment (flyflypeng) [inline wasm/service/kuasar-wasm.service:8]: Currently wasm type sandboxer doesn't support config reload feature, `ExecReload` needs to be removed.

Comment (Burning1020) [reply wasm/service/kuasar-wasm.service:8]: What is the consequence of keeping this `ExecReload`?

Comment (flyflypeng) [reply wasm/service/kuasar-wasm.service:8]: If the user runs the `systemctl reload wasm-sandboxer` command, the wasm-sandboxer process will receive the SIGHUP signal and process exit abnornally, which does not meet the expectation of the `systemctl reload` command.

Comment (jwcesign) [reply wasm/service/kuasar-wasm.service]: fixed

**Pull request merged**